eng/service-templates/ProductConstructionService/scheduledContainerJob.bicep

```diff
@@ -73,8 +73,9 @@ resource containerJob 'Microsoft.App/jobs@2024-03-01' = {
                   name: 'job'
                   env: env
                   command: [
-                      'dotnet'
-                      dllName
+                      '/bin/sh'
+                      '-c'
+                      'cd ./SubscriptionTriggerer && dotnet "${dllName}"'
```

Comment:
We're setting the WORKDIR to `ProductConstructionService` below, so we have to exit out of it first, before going to `SubscriptionTriggerer`

Reply:
Good catch